packages/docs-app/src/examples/core-examples/editableTextExamples.tsx

```diff
@@ -0,0 +1,58 @@
+/* !
+ * (c) Copyright 2024 Palantir Technologies Inc. All rights reserved.
+ */
+
+import dedent from "dedent";
+import * as React from "react";
+
+import { EditableText } from "@blueprintjs/core";
+import { CodeExample, type ExampleProps } from "@blueprintjs/docs-theme";
+
+export const EditableTextBasicExample: React.FC<ExampleProps> = props => {
+    const code = `<EditableText placeholder="Click to edit..." onConfirm={...} onCancel={...} />`;
+    return (
+        <CodeExample code={code} {...props}>
+            <EditableText
+                placeholder="Click to edit..."
+                // eslint-disable-next-line no-console
+                onConfirm={value => console.log("Confirmed:", value)}
```

Comment:
It would be nice to have these do something more visible. I played around with this example and didn't realize that it was logging to the console.

Reply:
Hm, we could use something like an alert, but I fear that might be a bit too disruptive. Do you have a suggestion on something more visible?

Reply:
Maybe a toast?

Reply:
Added in b12eedf5e42525cb6f4f5e812368c10f0845c00c / 2f97df674ae0db4680948912732ef9257ae8a9bc